### DISCUSSION TITLE
Download select results in a excel workbook

### DISCUSSION THREAD

**Discussion by amisano** [dbeaver/dbeaver#9228] [Q&A] [answered]
Hi to all
I'm newbi for this tool 
My question is: when I try to download the result of an sql selection to Excel, the choise Excel doesn't appear.
The only possibility is to download he selection to a Csv file.
This possibility exist ?
If exist, how have to do to install this option from this tool
![Immagine](https://user-images.githubusercontent.com/68107206/87143160-eb391300-c2a5-11ea-8acc-bba6b0d1d33f.jpg)
 ?
Many thanks in advance.

**Comment by kseniiaguzeeva** [comment, marked as answer]
You need to install MS Office extension.
1. Go to Help>Install new software...
2. Enter https://dbeaver.io/update/office/latest/ in **Work with** field
3. Check **DBeaver Office Support** in the field below
4. Go through the wizard
![dbeaver_XgN3jkcoTf](https://user-images.githubusercontent.com/51405061/87155047-f0588b00-c2c2-11ea-8947-9f0b3ea67556.png)
